app/_data/docs_nav_gateway_2.8.x.yml

```
@@ -114,6 +114,24 @@
           url: /plan-and-deploy/security/db-encryption
         - text: Kong Security Update Process
           url: /plan-and-deploy/security/kong-security-update-process
+        - text: Secrets Management
+          items:
+          - text: Getting Started
+            url: /plan-and-deploy/security/secrets-management/getting-started
+          - text: Advanced Usage
+            url: /plan-and-deploy/security/secrets-management/advanced-usage
+          - text: Backends Overview
+            url: /plan-and-deploy/security/secrets-management/backends-overview
+          - text: Backends
```

```suggestion
          - text: Backends
             url: /plan-and-deploy/security/secrets-management/backends-overview
```
No need to split the backends overview from the backends topics.